services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/LegStep.java

```diff
@@ -46,6 +46,17 @@ public static Builder builder() {
    */
   public abstract double duration();
 
+  /**
+   * The estimated travel time from the maneuver to the next {@link LegStep} including delay
+   * (road repair, traffic jam, etc.).
```

Comment:
```suggestion
   * The typical travel time for traversing this LegStep. There's a delay along the LegStep if you subtract this durationTypical() value from the LegStep duration() value and the resulting difference is greater than 0. The delay is because of any number of real-world situations (road repair, traffic jam, etc).
```